apps/prairielearn/src/pages/instructorQuestionPreview/instructorQuestionPreview.html.ts

```diff
@@ -47,13 +57,33 @@ export function InstructorQuestionPreview({ resLocals }: { resLocals: Record<str
       </div>
     `,
     content: html`
+      ${manualGradingPreviewEnabled
+        ? html`
+            <div class="alert alert-primary alert-dismissible fade show">
+              You are viewing this question as it will appear in the manual grading interface.
+              <a href="${normalPreviewUrl}" class="alert-link">Return to the normal view</a>.
+              <button
+                type="button"
+                class="btn-close"
+                data-bs-dismiss="alert"
+                aria-label="Close"
+              ></button>
```

Comment:
If you close this alert then I think there is no way in the UI to get back to the regular view. I would suggest making this alert non-closable, because otherwise we don't have a good way to indicate that you are in a non-normal view state.

Reply:
Another option is to have the dropdown button with the manual preview option be replaced with the same button and a checkmark before it, and that button then returns to regular view.

Reply:
Something in this direction (this is a crude POC with an emoji, you'd probably use a FA icon instead).
![image](https://github.com/user-attachments/assets/3e42ebfb-1c05-4a71-ac39-4c70707ab26b)

Reply:
I think I'll go the non-dismissible alert route for now. The styling to get checkboxes working in the user menu was pretty gnarly IIRC.

Reply:
Done in 66ddde1.